src/App.tsx

```diff
@@ -1,41 +1,72 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import './App.scss';
 import './styles/general.scss';
 import { PostsList } from './components/PostsList';
 import { PostDetails } from './components/PostDetails';
+import { User } from './react-app-env';
+import { Loader } from './components/Loader';
+import { getAllUsers } from './api/api';
 
-const App: React.FC = () => (
-  <div className="App">
-    <header className="App__header">
-      <label>
-        Select a user: &nbsp;
+const App: React.FC = () => {
+  const [allUsers, setAllUsers] = useState<User[]>([]);
+  const [currentUser, setCurrentUser] = useState('0');
+  const [selectedPostId, setSelectedPostId] = useState<number>();
```

Comment:
remove typing from a state with primitive values
```suggestion
  const [selectedPostId, setSelectedPostId] = useState();
```